src/applications/personalization/profile-2/routes.js

```diff
@@ -3,50 +3,55 @@ import PersonalInformation from './components/personal-information/PersonalInfor
 import MilitaryInformation from './components/MilitaryInformation';
 import DirectDeposit from './components/direct-deposit/DirectDeposit';
 import ConnectedApplications from './components/connected-apps/ConnectedApps';
+import { PROFILE_PATHS, PROFILE_PATH_NAMES } from './constants';
 
-const getRoutes = showDirectDeposit => {
-  const routes = [
+const getRoutes = config => {
```

Comment:
Also wondering if this `config` object should default to `{showDirectDeposit: true, showMilitaryInformation: true}` so that by default it just returns all the routes. Alternatively, the names of the flags could be reversed so that you need to pass in certain keys in order to remove certain routes. So if you passed in `{removeDirectDeposit: true}` you'd get back all the routes except Direct Deposit.

I could imagine a scenario where we add new optional routes to the Profile and now any calls to `getRoutes()` (I know it's just one call now!) would need to be modified so the new routes were returned by default.

Just thinking out loud here. Likely not worth worrying about too much until this function is used more widely!

Reply:
I like the approach of reversing the naming! Updated 👍  I also changed `config` to `options`.